vnext/Desktop.IntegrationTests/RNTesterIntegrationTests.cpp

```diff
@@ -197,6 +198,9 @@ TEST_CLASS (RNTesterIntegrationTests) {
 
 #pragma region Extended Tests
 
+  BEGIN_TEST_METHOD_ATTRIBUTE(Dummy)
+  TEST_IGNORE()
```

Comment:
Double-check the hang is a test issue and not a real regression, then link/comment an IOU issue to investigate getting it turned back on.